src/homepageExperience/components/steps/python/WriteData.tsx

```diff
@@ -46,7 +46,7 @@ export const WriteDataComponent = (props: OwnProps) => {
   useEffect(() => {
     setSelectedBucket(bucket)
     onSelectBucket(bucket.name)
-  }, [bucket])
+  }, [bucket, onSelectBucket])
```

Comment:
Final Q - for all of these, double check to make sure we're not still missing a dependency on any. (It would be annoying if you spent the time fixing these issues, and then there was one remaining on the same page!)